README.md

```diff
@@ -29,6 +43,17 @@ dotnet workload install aspire
 dotnet restore eShop.Web.slnf
 ```
 
+Or
+
+- Run the following commands in a Powershell & Terminal running as `Administrator` to automatically configuration your environment with the required tools to build and run this application. (Note: A restart is required after running the script below.)
+
+##### Install Visual Studio Code and related extensions
+```powershell
+install-Module -Name Microsoft.WinGet.Configuration -AllowPrerelease -AcceptLicense  -Force
```

Comment:
Why is the -Force required here?

Reply:
It avoid an interrupting prompt for acceptance that otherwise blocks until the user confirms in the CLI.